### PR TITLE
fix: Upgrade Next version and Metadata

### PR DESCRIPTION
**Pull request by Formasitchijoh**
## Summary
This pull request addresses the following tasks outlined  [here](https://docs.google.com/document/d/1iki5kQ0Daq9j42dwGsICiFZJxKeejqWpKFhp5FETLbM/edit):

- ###  Upgrade Template
 The project has been updated to the latest version of Next.js and its dependency, ensuring compatibility with the latest features and improvements provided by the framework.
- ###  Metadata updates
 No changes were made to the metadata as they were up-to-date.
- ###  Required changes
An error related to accessing undefined properties on navigating to the tutorial route has been fixed. Conditionally chaining the properties ensures a safe return in case a property is undefined.
- ###  Pull Request submission
 This pull request serves as a submission of the completed task, incorporating the upgrades, metadata updates, and required changes.
 
 @itzomen  Please kindly review when you are chanced and provide your feedback

### CODE REVIEW

File: .env
Comment (itzomen) [inline .env:1]: @Formasitchijoh Never commit environment files, the .env.example is a template used to create your env file so you do not expose secret variables like your API key.

- Delete this file
- Delete the API key and create a new one
- Do not commit environment files

File: package.json
Comment (itzomen) [inline package.json:1]: You added unnecessary dependencies, only update the dependencies that are present on the project

**Review by itzomen — CHANGES_REQUESTED** [review]

File: .env
Comment (Formasitchijoh) [reply .env:1]: Okay I'll remove that Sir

File: package.json
Comment (Formasitchijoh) [reply package.json:1]: Okay, I'll remove that

**Comment by Formasitchijoh** [comment]
@itzomen i made the required changes you requested,

**Review by itzomen — APPROVED** [review]

**Pull request merged**